Assets/Scripts/Quest/QuestSystem.cs

```diff
@@ -101,7 +101,7 @@ public class QuestSystem : MonoBehaviour
 		}
 		else {
 			// TODO: until we have a better idea, show the description as a message box for quest segments that aren't targeting a specific city (and so can't use the usual captain's log entry system which depends on city ids)
-			gameVars.ShowANotificationMessage(QuestMessageIntro + nextSegment.descriptionOfQuest);
+			if (segmentId != 0) { gameVars.ShowANotificationMessage(QuestMessageIntro + nextSegment.descriptionOfQuest); }
```

Comment:
Undo this, this was from your old opening cutscene stuff

Reply:
done